pkg/api/endpoints/util.go

```diff
@@ -28,6 +28,18 @@ import (
 	hashutil "k8s.io/kubernetes/pkg/util/hash"
 )
 
+const (
+	PodHostnameAnnotation  = "net.beta.kubernetes.io/podHostname"
```

Comment:
I would actually call this `pod.beta.kubernetes.io/hostname` - does it want to be in `pkg/api/pod/util.go`?